apps/studio/components/interfaces/TableGridEditor/TableGridEditor.tsx

```diff
@@ -188,11 +189,15 @@ const TableGridEditor = ({
   }
 
   const onSqlQuery = async (query: string) => {
-    const res = await meta.query(query)
```

Comment:
i'm actually not sure if onSqlQuery is being used actually - i can't find the trace inside SupabaseGrid which calls this method anywhere